job_executor/worker/steps/dataset_validator.py

```diff
@@ -39,8 +40,9 @@ def run_for_dataset(dataset_name: str) -> Tuple[Path, Path]:
             "Failed to validate dataset. "
             "Resolve errors with the microdata-validator before uploading."
         )
+    validated_path = save_validated_file(dataset_name)
```

Comment:
Why moving only the csv file?
This could be a oneliner:
```
validated_path = Path(WORKING_DIR / f"{dataset_name}.csv").rename(
   Path(WORKING_DIR / f"{dataset_name}_validated.csv")
)
```

Reply:
Yes a oneliner is all that is needed!